_site/css/style.css

```diff
@@ -0,0 +1,369 @@
+html, body {
+    min-height: 100% !important;
+    height: 100%;
+}
+nav {
+	margin-left: 5%;
+}
+
+::selection {
+	color: #FFF;
+	background-color: #ff3131;
+}
+
+-webkit-::selection {
+	color: #FFF;
+	background-color: #ff3131;
+}
+
+-msie-::selection {
+	color: #FFF;
+	background-color: #ff3131;
+}
+
+-moz-::selection {
+	color: #FFF;
+	background-color: #ff3243;
+}
+
+-o-::selection {
+	color: #FFF;
+	background-color: #ff3131;
+}
+
+img {
+	display: block;
+	width: 95%;
+	margin: 0 auto;
+	margin-top: 3%;
+}
+
+h2 {
+	font-family: 'Droid Sans';
+	text-align: center;
+}
+
+h3 {
+	text-align: left;
+	font-family: 'Droid Sans';
+}
+
+p {
+	font-family: "Times New Roman", Times, serif;
+}
+
+/*Strip the ul of padding and list styling*/
+ul {
+	list-style-type:none;
+	margin: 0 auto;
+	padding:0;
+	position: absolute;
+}
+
+/*Create a horizontal list with spacing*/
+li {
+	display:inline-block;
+	float: left;
+	margin-right: 1px;
+}
+
+/*Style for menu links*/
+li a {
+	display:block;
+	min-width:140px;
```

Comment:
@naveentherocker For properties use this syntax:
```prop: value;```
eg.
```min-width: 140px;```
Make these changes throughout the file

Reply:
Well I didn't add CSS styles. Everything you mentioned was already there. Ok I'll do those changes